frontend/amundsen_application/static/js/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,159 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import SanitizedHTML from 'react-sanitized-html';
+import { Modal } from 'react-bootstrap';
+
+import { IconSizes } from 'interfaces';
+import { NoticeSeverity } from 'config/config-types';
+import { AlertIcon, InformationIcon } from 'components/SVGIcons';
+import { DefinitionList } from 'components/DefinitionList';
+
+import './styles.scss';
+
+const SEVERITY_TO_COLOR_MAP = {
+  [NoticeSeverity.INFO]: '#3a97d3', // cyan50
+  [NoticeSeverity.WARNING]: '#ffb146', // $amber50
+  [NoticeSeverity.ALERT]: '#b8072c', // $red70
+};
+const SEVERITY_TO_SEVERITY_CLASS = {
+  [NoticeSeverity.INFO]: 'is-info',
+  [NoticeSeverity.WARNING]: 'is-warning',
+  [NoticeSeverity.ALERT]: 'is-alert',
+};
+const OPEN_PAYLOAD_CTA = 'See details';
+const PAYLOAD_MODAL_TITLE = 'Summary';
+const PAYLOAD_MODAL_CLOSE_BTN = 'Close';
+
+export interface AlertProps {
+  /** Message to show in the alert */
+  message: string | React.ReactNode;
+  /** Severity of the alert (info, warning, or alert) */
+  severity?: NoticeSeverity;
+  /** Link passed to set as the action (for routing links) */
+  actionLink?: React.ReactNode;
+  /** Text of the link action */
+  actionText?: string;
+  /** Href for the link action */
+  actionHref?: string;
+  /** Callback to call when the action is clicked */
+  onAction?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  /** Optional extra info to render in a modal */
+  payload?: Record<string, string>;
+}
+
+export const Alert: React.FC<AlertProps> = ({
+  message,
+  severity = NoticeSeverity.WARNING,
+  onAction,
+  actionText,
+  actionHref,
+  actionLink,
+  payload,
+}: AlertProps) => {
+  const [showPayloadModal, setShowPayloadModal] = React.useState(false);
+  let action: null | React.ReactNode = null;
+
+  const handleSeeDetails = (e: React.MouseEvent<HTMLButtonElement>) => {
+    onAction?.(e);
+    setShowPayloadModal(true);
```

Comment:
this is missing the logging logic in https://github.com/amundsen-io/amundsen/blob/main/frontend/amundsen_application/static/js/components/Alert/index.tsx